src/function/sort.ts

```diff
@@ -0,0 +1,30 @@
+import { Person } from "../types";
+import { SORT } from "../types/sort";
+import { SORT_DIRECTION } from "../types/sortDirection";
+
+export function sort(people: Person[], sortField: SORT, sortDirection: SORT_DIRECTION): Person[] {
+  if (sortDirection === SORT_DIRECTION.NONE) {
+      return [...people];
+  }
+
+  const multiplier = sortDirection === SORT_DIRECTION.ASC ? 1 : -1;
+
+  return [...people].sort((p1, p2) => {
+      switch (sortField) {
+          case SORT.NAME:
+              return p1.name.localeCompare(p2.name) * multiplier;
```

Comment:
The use of `localeCompare` for sorting names is appropriate for handling string comparisons. Ensure that the `name` property is always a valid string to avoid unexpected results.